cfme/common/vm_views.py

```diff
@@ -333,10 +334,14 @@ class form(BasicProvisionFormView):  # noqa
         cancel_button = Button('Cancel')
 
         def _select_template(self, template_name, provider_name):
-            try:
-                row = self.parent.image_table.row(name=template_name, provider=provider_name)
-                row.click()
-            except IndexError:
+            for _ in self.parent.paginator.pages():
+                try:
+                    self.parent.image_table.row(name=template_name, provider=provider_name).click()
```

Comment:
I think it was my bad; 
can you please try:
```
view.paginator.find_row_on_pages(view.image_table, name=template_name)
```

Let me know its works or not.

Reply:
@digitronik awesome, thanks! It works beautifully!

Reply:
Yay you used my function!